cmd/calyptia/create_aggregator_k8s.go

```diff
@@ -0,0 +1,134 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	apiv1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+
+	cloud "github.com/calyptia/api/types"
+)
+
+func newCmdCreateAggregatorOnK8s(config *config, testClientSet kubernetes.Interface) *cobra.Command {
+	var aggregatorName string
+	var noHealthCheckPipeline bool
+	var environmentKey string
+	var tags []string
+
+	loadingRules := clientcmd.NewDefaultClientConfigLoadingRules()
+	configOverrides := &clientcmd.ConfigOverrides{}
+
+	cmd := &cobra.Command{
+		Use:     "kubernetes",
+		Aliases: []string{"kube", "k8s"},
+		Short:   "Setup a new core instance on Kubernetes",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.Background()
+
+			var environmentID string
+			if environmentKey != "" {
+				var err error
+				environmentID, err = config.loadEnvironmentID(environmentKey)
+				if err != nil {
+					return err
+				}
+			}
+
+			created, err := config.cloud.CreateAggregator(ctx, cloud.CreateAggregator{
+				Name:                   aggregatorName,
+				AddHealthCheckPipeline: !noHealthCheckPipeline,
+				EnvironmentID:          environmentID,
+				Tags:                   tags,
+			})
+			if err != nil {
+				return fmt.Errorf("could not create core instance at calyptia cloud: %w", err)
+			}
+
+			if configOverrides.Context.Namespace == "" {
+				configOverrides.Context.Namespace = apiv1.NamespaceDefault
+			}
+
+			var clientset kubernetes.Interface
+			if testClientSet != nil {
+				clientset = testClientSet
+			} else {
+				kubeConfig := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(loadingRules, configOverrides)
+				kubeClientConfig, err := kubeConfig.ClientConfig()
+				if err != nil {
+					return err
+				}
+
+				clientset, err = kubernetes.NewForConfig(kubeClientConfig)
+				if err != nil {
+					return err
+				}
+
+			}
+
+			k8sClient := &k8sClient{
+				Interface:    clientset,
+				namespace:    configOverrides.Context.Namespace,
+				projectToken: config.projectToken,
+				cloudBaseURL: config.baseURL,
+				labelsFunc: func() map[string]string {
+					return map[string]string{
+						labelVersion:      version,
+						labelPartOf:       "calyptia",
+						labelManagedBy:    "calyptia-cli",
```

Comment:
I wonder if this should be say that it is managed by `calyptia-core` instead.
I mean, the pipelines will be.

CC @niedbalski @pwhelan